package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-with-direction",
-  "version": "1.3.0",
+  "version": "1.4.0",
```

Comment:
version numbers must never be bumped in PRs; separately, the deps should be a separate PR from this one. please revert these changes, especially the jest and babel ones.

Reply:
Reverted this changes